docs/pages/versions/unversioned/sdk/asset.md

```diff
@@ -14,13 +14,59 @@ import PlatformsSection from '~/components/plugins/PlatformsSection';
 
 <InstallSection packageName="expo-asset" />
 
-## API
+# API
 
 ```js
 import { Asset } from 'expo-asset';
 ```
 
-This class represents an asset in your app. It gives metadata about the asset (such as its name and type) and provides facilities to load the asset data.
+## Hooks
+
+### `useAssets`
+
+```ts
+const [assets, error] = useAssets([...]);
+```
+
+Download and store one or more assets locally. After the assets are loaded, with [`Asset.loadAsync`](#assetloadasyncmodules), this hook returns a list of asset instances.
```

Comment:
When we're describing a function, argument, class or whatever, we're talking about it in 3rd-person, so I think this would sound more natural 👇 

```suggestion
Downloads and stores one or more assets locally. After the assets are loaded, with [`Asset.loadAsync`](#assetloadasyncmodules), this hook returns a list of asset instances.
```

Reply:
Perfect, thanks 😄